sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -8,6 +8,20 @@
 
 ### Bugs Fixed
 
+- Handle injected "tenant_id" and "claims" ([#23138](https://github.com/Azure/azure-sdk-for-python/issues/23138))
+  
+  "tenant_id" argument in get_token() method is only supported by:
+
+  - `AuthorizationCodeCredential`
+  - `AzureCliCredential`
+  - `AzurePowerShellCredential`
+  - `InteractiveBrowserCredential`
+  - `DeviceCodeCredential`
+  - `EnvironmentCredential`
+  - `UsernamePasswordCredential`
+
+   it is omitted by other types of credentials.
```

Comment:
Nit: it might be a bit more accurate to say that we ignore the argument, but that's just my thought 🙂
```suggestion
   it is ignored by other types of credentials.
```